skimage/measure/profile.py

```diff
@@ -79,11 +87,15 @@ def profile_line(image, src, dst, linewidth=1,
     else:
         pixels = ndi.map_coordinates(image, perp_lines,
                                      order=order, mode=mode, cval=cval)
+    pixels = np.flip(pixels, axis=1)
```

Comment:
If anyone is curious about this, The outputted array with `linewidth=1` and `reduce_func=None` gives an array where the row (axis=1) values are flipped. Here I make this consistent. Tested in` test_reduce_func_None_linewidth_3()`.

Reply:
Great catch! That's could have let to confusing bugs down the line.

And I think your review comment would also make a great inline comment. ;) Secondly the flipping axis 1 is only necessary if `reduce_func is None` so I would move that statement into the corresponding if clause. Also maybe use `pixels[:, ::-1]` instead of `flip`? Its subjective but at least I'm more used to that syntax.

Reply:
Great I'll put in that clause!
I'm used to `np.flip` but I wonder which is more common in skimage? You call it.

By my review do you mean:

> The outputted array with linewidth=1 and reduce_func=None gives an array where the row (axis=1) values are flipped. Here I make this consistent.

Reply:
Yeah, I think an inline comment explaining the reason for reversing the last axis would be really useful.

Reply:
I'm more used to `pixels[:, ::-1]` but it doesn't really matter with a comment.

Reply:
> Secondly the flipping axis 1 is only necessary if reduce_func is None so I would move that statement into the corresponding if clause.

Just bringing this to your attention again.

Reply:
Actually scratch that! In case `reduce_func` operates on single values flipping the last axis is necessary. So it's fine as it is. :wink: